trieste/acquisition/sampler.py

```diff
@@ -329,6 +330,61 @@ def sample(self, at: TensorType, *, jitter: float = DEFAULTS.JITTER) -> TensorTy
         return mean[..., None, :, :] + tf.transpose(variance_contribution, new_order)
 
 
+class DeepGaussianProcessSampler(Sampler):
```

Comment:
it seems there is no simple way to make `IndependentReparametrizationSampler` class more flexible to allow for usage by DGPs - in that case, since this is really DGP specific sampler I would move it to a separate file in `models/gpflux/`

Reply:
Ok that makes sense